meinberlin/react/livequestions/QuestionModerator.jsx

```diff
@@ -1,190 +1,189 @@
-                }
-              >
-                <i className="fas fa-bookmark" aria-hidden="true" />
-              </button>
+            onClick={toggleIshidden}
+            aria-label={isHidden ? undoHiddenText : hiddenText}
+            title={isHidden ? undoHiddenText : hiddenText}
+          >
+            <i
+              className={classNames(
+                'fas fa-eye',
+                isHidden && 'fa-eye-slash'
+              )}
+              aria-hidden="true"
+            />
+          </button>
+        )}
+        {props.displayIsOnShortlist && (
+          <button
+            type="button"
+            className={classNames(
+              'cardbutton card__button card__button--shortlist',
+              isOnShortlist && 'card__button--active'
             )}
-            {this.props.displayIsAnswered && (
-              <button
-                type="button"
-                className={classNames(
-                  'cardbutton card__button card__button--answered',
-                  this.props.is_answered && 'card__button--active'
-                )}
-                onClick={this.toggleIsAnswered.bind(this)}
-                aria-label={doneText}
-                title={doneText}
-              >
-                <i className="fas fa-check" aria-hidden="true" />
-              </button>
+            onClick={toggleIsOnShortList}
+            aria-label={
+              isOnShortlist ? removeBookmarkText : addBookmarkText
+            }
+            title={
+              isOnShortlist ? removeBookmarkText : addBookmarkText
+            }
+          >
+            <i className="fas fa-bookmark" aria-hidden="true" />
+          </button>
+        )}
+        {props.displayIsAnswered && (
+          <button
+            type="button"
+            className={classNames(
+              'cardbutton card__button card__button--answered',
+              isAnswered && 'card__button--active'
             )}
-            {this.props.displayIsLive && (
-              <button
-                type="button"
-                className={classNames(
-                  'cardbutton card__button card__button--live',
-                  this.state.is_live && 'card__button--active'
-                )}
-                onClick={this.toggleIslive.bind(this)}
-                aria-label={this.state.is_live ? removeLiveText : addLiveText}
-                title={this.state.is_live ? removeLiveText : addLiveText}
-              >
-                <i className="fas fa-tv" aria-hidden="true" />
-              </button>
+            onClick={toggleIsAnswered}
+            aria-label={doneText}
+            title={doneText}
+          >
+            <i className="fas fa-check" aria-hidden="true" />
+          </button>
+        )}
+        {props.displayIsLive && (
+          <button
+            type="button"
+            className={classNames(
+              'cardbutton card__button card__button--live',
```

Comment:
can you check if `cardbutton` is still needed? I didnt find any styling for it. It also doesn't look very BEM to me.

Reply:
`cardbutton` is from the compiled berlin_css.css which is part of the Card HTML in the styleguide: https://styleguide.berlin.de/patterns/09-vertical_marketing-page-modul-card/09-vertical_marketing-page-modul-card.html